### PR TITLE
Toolbar headline sections

### PR DESCRIPTION
**Pull request by egichuhis**
Hello, Kindly review my PR.
Changes made:

- Add `index.html` and `style.css` files
- Add `README` and `LICENSE` files
- Install Linters: `Stylelint, Webhint, Lighthouse
- Add `linters.yml` config file for GitHub workflows
- Add `Header/Toolbar` and `Headline` sections
- Add social media section with icons

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+                    <li><a href="#"><i class="fa fa-github"></i></a></li>
+                    <li><a href="#"><i class="fa fa-linkedin"></i></a></li>
+                    <li><a href="#"><i class="fa fa-angellist"></i></a></li>
+                    <li><a href="#"><i class="fa fa-twitter"></i></a></li>
+                    <li><a href="#"><i class="fa fa-medium"></i></a></li>
+                </ul>
+            </span>
+        </section>
+
+    </body>
```
Comment (ichala) [inline index.html:63]: - [ ] to match the Figma design kindly fix those styling issues 🙏 :

1. kindly make sure ur header items are aligned to center 
2. kindly reduce the space from the left for the whole headline section
3. kindly update your headline title `Hey There. I'm Edwin` to match the design by reducing the text size and using the same font mentioned in Figma
4. the social media icons should be aligned to the left with the text on top of it , kindly fix that



your design           | figma
:-------------------------:|:-------------------------:
image | image

**Review by ichala — CHANGES_REQUESTED** [review]
Hi Student
Good job so far! 
👏👏👏 
You still need to work on some issues to go to the next project, but you're almost there!

the-simpsons-homer-simpson

## To Highlight 🎉

- No Linter errors  ✔️ 
- Good Design ✔️
- Clean Code ✔️


 ## Required Changes ♻️


- [ ] **_🛑  Check The Comments Below_**





## Optional suggestions

- N/A

**********
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by egichuhis** [comment]
Alignment of headline, toolbar and fonts fixed. Thanks.

**Review by Cabraham1 — APPROVED** [review]
Hi @egichuhis  , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**